zilliqa/src/node_launcher.rs

```diff
@@ -125,7 +125,7 @@ impl NodeLauncher {
 
         let rpc_module = api::rpc_module(Arc::clone(&node));
 
-        if !config.disable_rpc {
+        if !config.enabled_apis.is_empty() {
```

Comment:
After discussing with @DrZoltanFazekas , we think its important to be able to expose the API on more than one port with different configuration. I will make that change tomorrow.